EoT-Agents-Manifacturing-Marketplace/purchasing_agent_1/ethereum_private_key.txt

```diff
@@ -1 +1 @@
-e6e8981d75dc59538d78bdaeed0cfb7b1321358beeec415109ed54455ec9f024
\ No newline at end of file
+0x8f1bb6383b67a208bd73c2bc146167ef468d6a74d2e172d9c5b374a9d255c514
```

Comment:
Shall we create ticket to remove the private keys from the repo and provide documenation how to generate them conveniently?

Reply:
having some defined accounts makes it easier for https://github.com/boschresearch/open-eot-agents/issues/2 to set up a system with just a few commands. Therefore I think it is needed to have some predefined.
At least in the documentation all proper configurations need to be mentioned for adoption.
What do you think?

Reply:
Having private keys in the code base bears the risk, some adopter forgets to recreate their own, when creating something for production. I think right now it's no issue, but maybe for the future.